docs/pages/image-extractor.rst

```diff
@@ -5,7 +5,9 @@ Habitat Sim Image Extractor Tutorial
     :class: m-block m-default
 
 
-This notebook will go over how to use the Image Extraction API in Habitat Sim and the different user options available.
+This notebook will go over how to use the image extraction API in Habitat Sim and the different
```

Comment:
This is great, but could you give an example use-case of what someone might use an image extraction API for? (e.g. this notebook will go over how to extract images from Habitat-Sim to train semantic segmentation models)

Basically, think of the presenting solutions to problems rather than APIs for their own sake.